web/css/components/popup.css

```diff
@@ -0,0 +1,40 @@
+.container__popup {
+  display: flex;
+  align-items: flex-start;
+  flex-direction: column;
+  width: 784px;
+  padding: 40px;
+  gap: 8px;
+  border-radius: 24px;
+  background-color: #f2f6f9;
+}
+
+.container_img {
```

Comment:
The figure is the element that will restrict the image size. The img should be with:100% object-fit: fit